telebot/plugins/notify.py

```diff
@@ -0,0 +1,35 @@
+"""Send a direct message to user if user's name was mentioned in a group/channel.
+This plugin is to notification to user who is tagged. And this plugin need to
+have a configuration yaml file as template:
+
+For example:
+[
+    {"name": "@namnh307", "id": "473649555"},
+    {"name": "@daidv", "id": "432649532"}
+]
+"""
+
+import logging
+import json
+
+LOG = logging.getLogger(__name__)
+
+
+def check_messsage(users, message):
+    return [user['id'] for user in users if user['name'] in message]
+
+
+def handle(bot, update):
+    try:
+        config_user = json.load(open('/tmp/config_user.json'))
+    except FileNotFoundError:
+        LOG.warning('There is no the config_user.json at all')
+        return
+    user_ids = check_messsage(config_user, update.message.text)
```

Comment:
For now, there is only one line of code in this method. Why don't you move the line of code to here instead of calling "check_message".